app/src/main/java/com/sun/weather04/utils/Constant.java

```diff
@@ -0,0 +1,32 @@
+package com.sun.weather04.utils;
+
+public class Constant {
+    public static final String REQUEST_METHOD = "GET";
+    public static final int READ_TIMEOUT = 15000;
+    public static final int CONNECTION_TIMEOUT = 15000;
+    public static final String URL_BASE = "https://api.darksky.net/";
+    public static final String FORECAST = "forecast/";
+    public static final String KEY = "61d1f884da87a54a931d94d4070c2a27/";
+    public static final String LOCATION = "timezone";
+    public static final String CURRENTLY = "currently";
+    public static final String SUMMARY = "summary";
+    public static final String ICON = "icon";
+    public static final String TIME = "time";
+    public static final String HUMID = "humidity";
+    public static final String PRECIP_PROBABILITY = "precipProbability";
+    public static final String WIND_SPEED = "windSpeed";
+    public static final String TEMPERATURE = "temperature";
+    public static final String URL_CURRENT_WEATHER = URL_BASE + FORECAST + KEY;
+    public static final String CLEAR_DAY = "clear-day";
+    public static final String CLEAR_NIGHT = "clear-night";
+    public static final String RAIN = "rain";
+    public static final String SNOW = "snow";
+    public static final String SLEET = "sleet";
+    public static final String WIND = "wind";
+    public static final String FOG = "fog";
+    public static final String CLOUDY = "cloudy";
+    public static final String PARTLY_CLOUDY_DAY = "partly-cloudy-day";
+    public static final String PARTLY_CLOUDY_NIGHT = "partly-cloudy-night";
+    public static final String COORDINATES = "16.070818,108.216549";
```

Comment:
sao toạ độ lại set cứng thế e

Reply:
dạ vì hiện tại e chưa lấy được định vị a :))))